apps/mull-api/src/app/event/event.service.ts

```diff
@@ -18,6 +20,58 @@ export class EventService {
     return this.eventRepository.findOne(id);
   }
 
+  findHostEvents(userId: number): Promise<Event[]> {
+    return this.eventRepository.find({
+      relations: ['host'],
+      where: {
+        host: {
+          id: userId,
+        },
+      },
+    });
+  }
+
+  findCoHostEvents(userId: number): Promise<Event[]> {
+    return this.eventRepository
+      .createQueryBuilder('event')
+      .leftJoin('event.coHosts', 'user')
+      .where('user.id = :userId', { userId })
+      .getMany();
+  }
```

Comment:
Similar to my comment in the resolver
```suggestion
  findHostedEvents(userId: number): Promise<Event[]> {
    return this.eventRepository.find({
      relations: ['host'],
      where: {
        host: {
          id: userId,
        },
      },
    });
  }

  findCoHostedEvents(userId: number): Promise<Event[]> {
    return this.eventRepository
      .createQueryBuilder('event')
      .leftJoin('event.coHosts', 'user')
      .where('user.id = :userId', { userId })
      .getMany();
  }
```